thunder/tests/test_grad.py

```diff
@@ -239,8 +239,16 @@ def check_jvp(f, *primals, comp, executor):
         AssertionError: If the Jacobian-vector product is not correct.
     """
     tangents = tree_map(make_tensor_like, primals)
-    actual_p, actual_t = executor.make_callable_legacy(jvp(f))(primals, tangents)
-    expected_p, expected_t = numerical_jvp(executor.make_callable_legacy(f))(primals, tangents)
+    initial_trace_jvp_f = thunder.trace()(jvp(f), primals, tangents)
+    actual_p, actual_t = executor.make_callable(initial_trace_jvp_f.python_callable(), disable_torch_autograd=True)(
+        primals, tangents
+    )
+
+    # dirty little trick for speed: skip the prologue
+    jf = executor.make_callable(f, disable_torch_autograd=True)
+    comp_f = thunder.compile_data(jf).get_computation_and_inputs(*primals)[0].computation_fn
```

Comment:
skipping the prologue here (and reducing the number of cases for polygamma) made things on the windows runner go from >= 35 min to ~28:30 min.